modules/vaos/app/services/vaos/jwt.rb

```diff
@@ -2,6 +2,13 @@
 
 module VAOS
   class JWT
+    VERSION = 2.1
+    ISS = 'gov.va.vaos'
+    ID_TYPE = 'ICN'
+    AUTHORITY = 'gov.va.iam.ssoe.v1'
```

Comment:
Would like to see some docs on what these constants represent? Maybe that's covered elsewhere or could go into a top-level class docs

Reply:
I can link to this but I think the documentation lives in coderepo.va.mobile. With their permission I might be able to expose that documentation.